lab_6_pipeline/pipeline.py

```diff
@@ -128,12 +244,22 @@ def convert_pos(self, tags: OpencorporaTagProtocol) -> str:  # type: ignore
         """
         Extracts and converts POS from the OpenCorpora tags into the UD format
         """
+        return self._tag_mapping[self.pos][tags.POS or 'UNKN']
 
     def convert_morphological_tags(self, tags: OpencorporaTagProtocol) -> str:  # type: ignore
         """
         Converts the OpenCorpora tags into the UD format
         """
+        parsed_tags = {attr: getattr(tags, attr, None) for
+                       attr in ['case', 'number', 'gender', 'animacy']}
```

Comment:
I think that tagnames are also defined in a parent

Reply:
fixed